pycord/ext/gears/gear.py

```diff
@@ -20,22 +20,24 @@
 # SOFTWARE
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Type, TypeVar, Generic
+from types import SimpleNamespace
+from typing import TYPE_CHECKING, Any, Generic, Type, TypeVar
 
 from ...commands import Command, Group
 from ...types import AsyncFunc
-from types import SimpleNamespace
 
 if TYPE_CHECKING:
     from ...bot import Bot
 
 
 T = TypeVar('T')
 
+
 class BaseContext(SimpleNamespace):
     ...
 
-ContextT = TypeVar("ContextT", bound=BaseContext)
+
+ContextT = TypeVar('ContextT', bound=BaseContext)
```

Comment:
Reverts out of scope change.
```suggestion
ContextT = TypeVar('ContextT', bound=BaseContext)
```